vep/__init__.py

```diff
@@ -25,53 +30,68 @@ def add_cli_arguments(self, parser):
 
         group.add_argument(
             '--package-prefix',
-            default = '/usr/local',
-            help = "Path to prefix the entire package with"
+            default='/usr/local',
+            help="Path to prefix the entire package with"
         )
 
         group.add_argument(
             '--repo-url',
-            default = pycmd('setup.py', '--url').strip(),
-            help = "Repo URL to pass through to fpm"
+            default=pycmd('setup.py', '--url').strip(),
+            help="Repo URL to pass through to fpm"
         )
 
         group.add_argument(
             '--package-name',
-            default = pycmd('setup.py', '--name').strip(),
-            help = "The package name, as seen in apt"
+            default=pycmd('setup.py', '--name').strip(),
+            help="The package name, as seen in apt"
         )
 
         group.add_argument(
             '--package-version',
-            default = pycmd('setup.py', '--version').strip(),
-            help = "The package version."
+            default=pycmd('setup.py', '--version').strip(),
+            help="The package version."
+        )
+
+        group.add_argument(
+            '--skip-scripts',
+            default=False,
+            action='store_true',
+            help="Set this to skip installing all the scripts in all the setup.py files in all the requirements"
```

Comment:
wording nit: most unix commands would phrase this as "Skip installing all the..."